main.go

```diff
@@ -21,13 +21,16 @@ func main() {
 	var httpServerConfig baseapp.HTTPConfig
 	pflag.StringVar(&httpServerConfig.Address, "http-address", "localhost", "http listen address")
 	pflag.IntVar(&httpServerConfig.Port, "http-port", 8080, "http listen port")
-	pflag.StringVar(&httpServerConfig.PublicURL, "http-public_url", "https://localhost:8080", "http public url")
+	pflag.StringVar(&httpServerConfig.PublicURL, "http-public-url", "https://localhost:8080", "http public url")
 
 	var githubappConfig githubapp.Config
-	pflag.StringVar(&githubappConfig.V3APIURL, "github-v3_api_url", "https://api.github.com/", "github v3 api url")
-	pflag.Int64Var(&githubappConfig.App.IntegrationID, "github-integrationID", 0, "github App ID (App->General->About->App ID)")
-	pflag.StringVar(&githubappConfig.App.WebhookSecret, "github-webhookSecret", "", "github webhook secret")
-	pflag.StringVar(&githubappConfig.App.PrivateKey, "github-privateKey", "", "github app private key content")
+	pflag.StringVar(&githubappConfig.V3APIURL, "github-v3-api-url", "https://api.github.com/", "github v3 api url")
+	pflag.Int64Var(&githubappConfig.App.IntegrationID, "github-integration-id", 0, "github App ID (App->General->About->App ID)")
+	pflag.StringVar(&githubappConfig.App.WebhookSecret, "github-webhook-secret", "", "github webhook secret")
+	pflag.StringVar(&githubappConfig.App.PrivateKey, "github-private-key", "", "github app private key content")
+
+	var messageTemplate string
+	pflag.StringVar(&messageTemplate, "message-template", "'{{.Branch}}' will auto-release to: {{range .AutoReleaseEnvironments}}\n {{.}}{{end}}", "Template string used when commenting on pull requests on Github. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]. See template.go for usage.")
```

Comment:
From a operations perspective I think the last sentence is redundant, so maybe we can drop that?

```suggestion
	messageTemplate := pflag.String("message-template", "'{{.Branch}}' will auto-release to: {{range .AutoReleaseEnvironments}}\n {{.}}{{end}}", "Template string used when commenting on pull requests on Github. The template format is Go templates [http://golang.org/pkg/text/template/#pkg-overview].")
```

Further more this could be `pflag.String` like the auth token and RM URL flags, that does however introduce a pointer dereference in its usage like the other flag variables.

Reply:
Changes accepted